docs/features/multiple-networks/multi-homing.md

```diff
@@ -110,6 +112,11 @@ spec:
 - `excludeSubnets` (string, optional): a comma separated list of CIDRs / IPs.
   These IPs will be removed from the assignable IP pool, and never handed over
   to the pods.
+- `disablePortSecurity` (boolean, optional): enable this flag to opt-out of port
+  security, thus removing MAC / IP spoofing protection from your workloads.
+- `allowL2Unknown` (boolean, optional): enable this flag to allow traffic for
+unknown destinations to be delivered to the ports attached to the secondary
+network.
```

Comment:
Do we have an appropriate place here or in some other doc on what would be the potential use cases of these options? Like nested virtualization?